### ISSUE TITLE
Collection.basicDashboard does nothing

### ISSUE THREAD

**Issue by saadfarooq** [deployd/deployd#503]
I tried to add an object to my `CustomCollection`'s `basicDashboard` property like so:

     CustomCollection.basicDashboard = {
      settings: [{
          name: 'propertyName',
          type: 'text',
          description: "This description appears below the text field"
      }, {
          name: 'longTextProperty',
          type: 'textarea'
      }, {
          name: 'numericProperty',
          type: 'number'
      }, {
          name: 'booleanProperty',
          type: 'checkbox'
      }]
    };

but nothing happens. This seems to work fine with custom resources though. Anyone know why ?

**Comment by ericfong** [comment]
code look ok to me. can we know more about other codes?

**Comment by saadfarooq** [comment]
It's basically a copy of UserCollection with that added. What is basicDashboard supposed to do anyway and where does it show up ?

**Comment by ericfong** [comment]
Here is one of the sample usage of basicDashboard
https://github.com/deployd/dpd-email/blob/master/index.js

It will build a 'config' page in ```/dashboard/email```

**Comment by saadfarooq** [comment]
It works fine with Resources. I was wondering how to get it to work with Collections. I assume that since there is a settings page in collection, that it would show up there but nada.

**Comment by ericfong** [comment]
It is not designed for Collections. May be that is a bug

**Comment by saadfarooq** [comment]
Yeah. The docs should it as Collection.basicDashboard.

At the very least, that should be fixed. 

But it still begs the question, what is the settings page in collections for ?

**Comment by ericfong** [comment]
setting page for collection is the properties page

**Comment by saadfarooq** [comment]
Oh sorry... I screwed that up. So essentially we can't add configuration properties to collections at the moment. By configuration properties, I mean things like SALT's, secrets, etc.

**Comment by andreialecu** [comment]
@saadfarooq You can use the dashboard to add a `salt` column to the UserCollection, which you can then store on a per record basis. The UserCollection is the same things as a Collection but it has a few extra methods.

Can you explain what it is you need that is not covered by the existing API?

**Comment by saadfarooq** [comment]
Yeah... I think that would work. It just seems semantically weird to do that in UserCollection when Collection seems to indicate that it should do it.
But feel free to close, I got my answer.
